### PR TITLE
service-page-html

### PR DESCRIPTION
**Pull request by Mzojobe**
please review service page for HTML

### CODE REVIEW

**Review by Vusumzi-qod — APPROVED** [review]

File: pages/service.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                    <li><a href="#">ABOUT US</a></li>
+                </ul>
+            </nav>
+    </div>
+    <div>
+        <img src="../images/ia-logo-orange-full.png.png" height="50px">
+    </div>
+    <main>
+        <div>
+            <h1>SERVICE</h1>
```
Comment (qodvilla) [inline pages/service.html]: This should be SERVICES

**Review by qodvilla — CHANGES_REQUESTED** [review]

File: pages/service.html (hunk 1 above)
Comment (Mzojobe) [reply pages/service.html]: okay !

**Review by qodvilla — APPROVED** [review]

File: pages/service.html [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/remixicon@2.5.0/fonts/remixicon.css" rel="stylesheet">
+    <title>Service</title>
+</head>
+<body>
+</head>
```
Comment (codedilla) [inline pages/service.html]: please remove the head closing tag

File: pages/service.html [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+                <ul>
+                    <li><a href="#">INSPIRE BOOKS</a></li>
+                    <li><a href="#">22 CENTURY EDUCATION</a></li>
+                    <li><a href="#">TUTORING</a></li>
+                    <li><a href="#">SERVICES</a></li>
+                    <li><a href="#">ABOUT US</a></li>
+                </ul>
+            </nav>
+    </div>
+    <div>
```
Comment (codedilla) [inline pages/service.html:48]: please wrap this <div> inside the <main> tag also

File: pages/service.html [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+        <img src="../images/ia-logo-orange-full.png.png" height="50px">
+    </div>
+    <main>
+        <div>
```
Comment (codedilla) [inline pages/service.html:52]: please wrap this with section inside the main tag as well

**Review by codedilla — CHANGES_REQUESTED** [review]

File: pages/service.html (hunk 3 above)
Comment (Mzojobe) [reply pages/service.html:48]: okay!

File: pages/service.html (hunk 2 above)
Comment (Mzojobe) [reply pages/service.html]: okay

**Review by sonwabi — APPROVED** [review]

**Review by qod-dante — APPROVED** [review]

**Review by codedilla — APPROVED** [review]

**Pull request merged**